### ISSUE TITLE
Locales aren't migrated to sites as expected

### ISSUE THREAD

**Issue by FrittenKeeZ** [statamic/migrator#90]
In v2 our `site/settings/system.yaml` has these locales:
```xml
locales:
  da:
    name: Danish
    full: da_DK
    url: "{env:APP_URL}/"
  en:
    name: English
    full: en_US
    url: "{env:APP_URL}/en/"
  sv:
    name: Swedish
    full: sv_SE
    url: "{env:APP_URL}/sv/"
```

In v3 these are migrated to `config/statamic/sites.php` as:
```php
    'sites' => [

        'default' => [
            'name' => 'Danish',
            'locale' => 'da_DK',
            'url' => '{env:APP_URL}/',
        ],

        'en' => [
            'name' => 'English',
            'locale' => 'en_US',
            'url' => '{env:APP_URL}/en/',
        ],

        'sv' => [
            'name' => 'Swedish',
            'locale' => 'sv_SE',
            'url' => '{env:APP_URL}/sv/',
        ],

    ],
```
I'd expect the key for Danish to be `da` instead of `default`.

A nice addition and helping hand, could be to convert the `{env:APP_URL}` part of URLs:
```php
$url = preg_replace('/{env:([A-Z0-9_]+)}/', '{env(\'$1\')}', $url);
```

**Comment by jasonvarga** [comment]
Maybe you missed renaming the tree file?

**Comment by FrittenKeeZ** [comment]
Doesn't seem like it - the site just responds with a 404 error
![image](https://user-images.githubusercontent.com/1186125/131862329-55927120-1afc-4387-b069-a95d01260c05.png)

Globals has also been renamed

**Comment by jasonvarga** [comment]
Where's the homepage? In the `pages` collection? (Probably)
Can you show `content/collections/pages.yaml`?

**Comment by FrittenKeeZ** [comment]
That's it! I missed the collection structure changes 🙌

**Comment by jasonvarga** [comment]
We dug into this. Looks like we intentionally change the first site to `default`. It was for a reason, but we can't find the reason, yet. 😆

**Comment by FrittenKeeZ** [comment]
I guess it makes sense for single sites, but I'd imagine most multi sites use the key/handle for language selectors (with icons) like we do 🤷‍♂️

**Comment by FrittenKeeZ** [comment]
@jesseleite any update on this one?

**Comment by jesseleite** [comment]
@FrittenKeeZ Will dig into this one again soon 👍 I can't remember the reason, but leaving this issue open until we take another look.

Curious though, can you change the site handle from `default` to your preferred handle post-migration? Should be a relatively easy find-and-replace in your project I would think?

**Comment by FrittenKeeZ** [comment]
@jesseleite yes I'm doing that during my testing, though it's easy to overlook various configurations and folders.

**Comment by FrittenKeeZ** [comment]
@jesseleite I just took a look at this, and since v3 works quite fine without using `default` it all comes down to convenience of migrating content. The files affecting the conversion to `default` are these:
```
trait Statamic\Migrator\Concerns\MigratesLocalizedContent
class Statamic\Migrator\GlobalSetMigrator
class Statamic\Migrator\PagesMigrator
class Statamic\Migrator\SettingsMigrator
```

Specifically it seems like you're using `default` for the base content, instead of dynamically pulling the first site's key instead, so it should be possible to use the original key all the way through.
We'll be using this tool for 7 sites, 3 of which are multi-language setups, so it would be nice not having to do manual handling every time we test the migration.
